### PR TITLE
LR-153. Pagination

### PR DESCRIPTION
**Pull request by anatoliiibrahimov**
### JIRA:
LR-153.Pagination: http://ssu-jira.softserveinc.com/browse/LR-153

### Reminder:
If there is checkbox near your nickname, please:
* Check code and changes;
* Leave your comments, share your thoughts with others;
* If you don't have any comments/ideas, please check checkbox that everything is OK.
(If you are last reviewer, please write to merge duty responsible (Gitter, whatever), that he could megre branches)

### Code reviewers for this pull request:
- [x] @vovka 
- [x] @superNazo 
- [x] @sergiy17

### CODE REVIEW

**Comment by vovka** [comment]
looks good

**Comment by vovka** [comment]
Tests failed

**Comment by brouhaha88** [comment]
STOP PUSHING !

**Comment by anatoliiibrahimov** [comment]
@vovka now everything is ok

**Comment by vovka** [comment]
Remove coverage directory please.

**Comment by anatoliiibrahimov** [comment]
@vovka deleted

**Comment by sergiy17** [comment]
continuous-integration/codeship — Build failed

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,6 @@
 
 # Ignore application configuration
 /config/application.yml
+
+# Ignore coverage folder
+/coverage
```
Comment (vovka) [inline .gitignore]: I believe this shoud be removed

**Pull request merged**